beacon-chain/state/setters.go

```diff
@@ -512,7 +525,9 @@ func (b *BeaconState) AppendPreviousEpochAttestations(val *pbp2p.PendingAttestat
 	b.lock.RLock()
 	atts := b.state.PreviousEpochAttestations
 	if b.sharedFieldReferences[previousEpochAttestations].refs > 1 {
-		atts = b.PreviousEpochAttestations()
+		copiedAtts := make([]*pbp2p.PendingAttestation, len(atts), len(atts)+1)
```

Comment:
Here too, why +1

Reply:
Because we are appending a value below, so setting the capacity + 1 will prevent us from having to copy the whole slice when appending the value